src/sdk.js

```diff
@@ -0,0 +1,72 @@
+import Web3 from 'web3';
+import IPFSClient from 'ipfs-http-client';
+import url from 'url';
+import RegistryNetworks from 'singularitynet-platform-contracts/networks/Registry.json';
+import RegistryAbi from 'singularitynet-platform-contracts/abi/Registry.json';
+
+import Account from './Account';
+import MPEContract from './MPEContract';
+import ServiceClient from './ServiceClient';
+import { find } from 'lodash';
+
+const DEFAULT_GAS_LIMIT = 210000;
+const DEFAULT_GAS_PRICE = 4700000;
+
+export default class SnetSDK {
+  constructor(config) {
+    this._config = config;
+    const options = {
+      defaultGas: this._config.defaultGasLimit || DEFAULT_GAS_LIMIT,
+      defaultGasPrice: this._config.defaultGasPrice || DEFAULT_GAS_PRICE,
+    };
+    this._networkId = config.networkId;
+    this._web3 = new Web3(config.web3Provider, null, options);
+    this._mpeContract = new MPEContract(this._web3, this._networkId);
+    this._account = new Account(this._web3, this._networkId, this._config, this._mpeContract);
+    this._web3.eth.defaultAccount = this._account.address;
+    const registryAddress = RegistryNetworks[this._networkId].address;
+    this._registryContract = new this._web3.eth.Contract(RegistryAbi, registryAddress, { from: this._account.address });
+  }
+
+  get account() {
+    return this._account;
+  }
+
+  get mpeContract() {
+    return this._mpeContract;
+  }
+
+  get web3() {
+    return this._web3;
+  }
+
+  get blockOffset() {
+    return this._config.blockOffset;
```

Comment:
Could you please clarify what does `blockOffset` mean here?

Reply:
This offset would be used to add to the default payment channel expiration [here](https://github.com/singnet/snet-sdk-js/pull/1/files?file-filters%5B%5D=.js#diff-622caf8e642effe65230813d19935735R65).

Reply:
Ah, got it. As I wrote before I think that calculation which `defaultChannelExpiration` does looks for me as part of payment channel strategy implementation. And this parameter looks like "channel expiration time reserve" or "margin". I would move this as configuration parameter to the strategy class.

Reply:
So what you mean to say is this could be specific to individual strategy as opposed to being generic at SDK level? If so then we could remove it from the config itself and let the strategy decide what the margin should be.

Reply:
:+1: Yes, that is what I meant, we actually could remove it from the config and make it parameter of strategy.

Reply:
Addressed in [commit](https://github.com/singnet/snet-sdk-js/pull/1/commits/b5f5df69677c9b97be4dc94d450b3c5876b9e670)